### DISCUSSION TITLE
Having trouble changing the calculation method to left

### DISCUSSION THREAD

**Discussion by Notexe** [klaasnicolaas/home-assistant-glow#561] [Q&A] [answered]
I've tried to follow the instructions on https://glow-energy.io/docs/faq/abrupt_consumption_with_solar/ by adding the configuration into the yaml for my device in ESPHome. 

Although I'm getting the following validation error in ESPHome's yaml editor:
`[method] is an invalid option for [total]. Please check the indentation.`
![image](https://github.com/user-attachments/assets/8c0b21d8-b07a-4f01-a288-f6aef5ea059e)

Is there any other way to do this? I'm kind of stumped at the moment. I was reading the changelog and apparently the full config is meant to import if you adopt the device in ESPHome but that didn't seem to happen to me for some reason.

I hope that someone will be able to help and thanks.

**Comment by klaasnicolaas** [comment, marked as answer]
The example is not correct, it only works on the daily sensor and not on the total sensor. I will adjust this today.

./Klaas

**Reply by Notexe** [reply]
Oh, thanks for getting back so quickly.

**Reply by seanq93** [reply]
Hey Klaas, did you ever update this? I'm trying to make the change today and I'm having the same issue as Notexe.

**Reply by Notexe** [reply]
It works fine for me with the latest change to the documentation.

This is what I have in my config:
```
sensor:
  - id: !extend sensor_total_daily_energy
    method: left
```

**Reply by klaasnicolaas** [reply]
It has been adjusted in the documentation, took a while until a new release before it came online 😉 

./Klaas

**Comment by seanq93** [comment]
Thanks, copy pasting an hour ago broke my configuration but works now after
a clean install.
